cromulent/vocab.py

```diff
@@ -238,7 +238,7 @@ def register_instance(name, data):
 
 	"Architecture": {"parent": HumanMadeObject, "id":"300263552", "label": "Architecture"},
 	"Armor": {"parent": HumanMadeObject, "id":"300036745", "label": "Armor"},
-	"Book": {"parent": HumanMadeObject, "id":"300028051", "label": "Book"},
+	"BookThing": {"parent": HumanMadeObject, "id":"300028051", "label": "Book"},
```

Comment:
Does this change mean the common type mapping in pipeline needs to change all occurrences of `Book` to `BookThing`?